docs/en/stack/ml/df-analytics/ml-feature-importance.asciidoc

```diff
@@ -0,0 +1,35 @@
+[role="xpack"]
+[[ml-feature-importance]]
+=== Understanding results: {feat-imp}
+
+{feat-imp-cap} is calculated for supervised {ml} methods such as 
+<<dfa-regression,{regression}>> and <<dfa-classification,{classification}>>. 
+This value provides further insight into the results of a {dfanalytics-job} and 
+therefore helps interpret these results. As we mentioned, there are multiple 
+features of a data point that are analyzed during {dfanalytics}. These features 
+are responsible for a particular prediction to varying degrees. {feat-imp-cap} 
+shows to what degree a given feature of a data point contributes to the 
+prediction. The feature importance value of a feature can be either positive or 
+negative depending on its effect on the prediction. If the feature reduces the 
+prediction value, the value is negative. If the feature increases the 
+prediction, the {feat-imp} value positive. The magnitude of the {feat-imp} 
+value shows how significantly the feature affects the prediction both locally 
+(for a given data point) or generally (for the whole data set).
+
+{feat-imp-cap} in the {stack} is calculated using the SHAP (SHapley Additive 
+exPlanations) method as described in
+https://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions.pdf[Lundberg, S. M., & Lee, S.-I. A Unified Approach to Interpreting Model Predictions. In NeurIPS 2017].
+
+By default, {feat-imp} values are not calculated. To generate this information, 
+when you create a {dfanalytics-job} you must specify the 
+`num_top_feature_importance_values` property. The {feat-imp} values are stored 
+in the destination index in fields prefixed by `ml.feature_importance`.
```

Comment:
> in fields prefixed by `ml.feature_importance`:

I think this might have changed, since in 7.x I'm seeing that information in an array inside the results for each document, like this:

> "ml" : {
            "FlightDelayMin_prediction" : 4.357336521148682,
            "feature_importance" : [
              {
                "feature_name" : "DistanceMiles",
                "importance" : 15.865198439768983
              },
              {
                "feature_name" : "FlightTimeMin",
                "importance" : -69.57508138095754
              }
            ],